receive-sqs.sh

```diff
@@ -1,57 +1,52 @@
 #!/bin/bash
 
-# reads a message from an SQS named SQS_QUEUE_NAME and performs the update based on the .version field of the message
+source .env-cron
 
 if [ -n "${SQS_QUEUE_NAME}" ]; then
   SQS_URL=$(aws sqs get-queue-url --queue-name "${SQS_QUEUE_NAME}" | jq -r .QueueUrl)
-
-  # Update the version if we have messages in the SQS
-  if [ -n "${SQS_URL}" ]; then
-    MSG=$(aws sqs receive-message --queue-url "$SQS_URL");
-
-    if [ -n "$MSG" ]; then
-      export DOCKER_TAG
-      export TARGETED_REGION
-      export EXIT_CODE
-
-      DOCKER_TAG=$(echo "$MSG" | jq -r '.Messages[0].Body' | jq -r .Message | jq -r .version);
-      TARGETED_REGION=$(echo "$MSG" | jq -r '.Messages[0].Body' | jq -r .Message | jq -r .region);
-
-      if [ -n "$TARGETED_REGION" ] && [ "$DEPLOYMENT_REGION" != "$TARGETED_REGION" ]; then
-       exit 0
+  MSG=$(aws sqs receive-message --queue-url $SQS_URL)
+  if [ ! -z "$MSG" ]
+  then
+    COUNT=$(echo $MSG | jq ".Messages|length")
+    if [ $COUNT > 0 ]
+    then
+      RECEIPT=$(echo $MSG | jq ".Messages[0].ReceiptHandle")
+      aws sqs delete-message --queue-url $SQS_URL --receipt-handle "$RECEIPT"
+      BODY=$(echo $MSG | jq ".Messages[0].Body | fromjson")
+      DOCKER_TAG_CONTENT=$(echo $BODY | jq -r '.tag')
+      DOCKER_TAG_LIGHTOUSE=$(echo $BODY | jq -r '.lighthouse');
+      DOCKER_TAG_EXPLORER=$(echo $BODY | jq -r '.explorerbff');
+
+      WAIT=$(echo $BODY | jq -r '.wait')
+      cd /opt/ebs/catalyst-owner
+      # Content docker tag
+      if grep -P '^#?DOCKER_TAG.*' .env; then
+        sed -i "s/^#\?DOCKER_TAG.*/DOCKER_TAG=$DOCKER_TAG_CONTENT/g" .env
```

Comment:
can you replace the `sed` by `echo >>` to append at the end? so we don't lose the history in case of manual rollback